.coveragerc

```diff
@@ -1,6 +1,11 @@
 [run]
 source = openapi
-omit = openapi/reload.py
+omit =
+    openapi/reload.py
+    openapi/db/migrations.py
```

Comment:
why are you omitting these below?

Reply:
They seem more operational code. I don't think it's worth to add tests for those